condition.go

```diff
@@ -0,0 +1,84 @@
+package nanny
+
+import (
+	"fmt"
+	"regexp"
+	"strconv"
+	"strings"
+)
+
+type RecordCondition struct {
+	Name    string
+	Enabled bool
+	Path    string
+	Value   string
+}
+
+func (c RecordCondition) String() string {
+	return c.Name
+}
+
+// Matches returns true if the event matches the specification of this condition.
+func (r RecordCondition) Matches(ev Event) bool {
+	if !r.Enabled {
+		return true
+	}
+	tokens := strings.Split(r.Path, ".")
+	if len(tokens) == 0 {
+		return false
+	}
+	// first is field of event
+	switch tokens[0] {
+	case "level":
+		return strings.ToLower(ev.Level.String()) == strings.ToLower(r.Value)
+	case "message":
+		// is it a pattern?
+		if strings.Contains(r.Value, "*") {
```

Comment:
Replace test for * with test for /<something>/ to allow * to be used inside condition value.